client/i18n/locales/chinese/translations.json

```
@@ -1,444 +1,444 @@
-    "msg-17": "Username is already associated with a different account",
-    "msg-18": "We have updated your username to {{username}}",
-    "msg-19": "We could not log you out, please try again in a moment",
-    "msg-20": "The email encoded in the link is incorrectly formatted",
-    "msg-21": "Oops, something is not right, please request a fresh link to sign in / sign up",
-    "msg-22": "Looks like the link you clicked has expired, please request a fresh link, to sign in",
-    "msg-23": "Success! You have signed in to your account. Happy Coding!",
-    "msg-24": "We are moving away from social authentication for privacy reasons. Next time we recommend using your email address: {{email}} to sign in instead.",
-    "msg-25": "We need your name so we can put it on your certification. Add your name to your account settings and click the save button. Then we can issue your certification.",
-    "msg-26": "It looks like you have not completed the necessary steps. Please complete the required projects to claim the {{name}} Certification.",
-    "msg-27": "It looks like you already have claimed the {{name}} Certification",
-    "msg-28": "@{{username}}, you have successfully claimed the {{name}} Certification! Congratulations on behalf of the freeCodeCamp.org team!",
-    "msg-29": "Something went wrong with the verification of {{name}}, please try again. If you continue to receive this error, you can send a message to support@freeCodeCamp.org to get help.",
-    "msg-30": "Error claiming {{certName}}",
-    "msg-31": "We could not find a user with the username \"{{username}}\"",
-    "msg-32": "This user needs to add their name to their account in order for others to be able to view their certification.",
-    "msg-33": "This user is not eligible for freeCodeCamp.org certifications at this time.",
-    "msg-34": "{{username}} has chosen to make their portfolio private. They will need to make their portfolio public in order for others to be able to view their certification.",
-    "msg-35": "{{username}} has chosen to make their certifications private. They will need to make their certifications public in order for others to be able to view them.",
-    "msg-36": "{{username}} has not yet agrees to our Academic Honesty Pledge.",
-    "msg-37": "It looks like user {{username}} is not {{cert}} certified",
-    "msg-38": "That does not appear to be a valid challenge submission",
-    "msg-39": "You have not provided the valid links for us to inspect your work.",
-    "msg-40": "No social account found",
-    "msg-41": "Invalid social account",
-    "msg-42": "No {{website}} account associated",
-    "msg-43": "You've successfully unlinked your {{website}}",
-    "msg-44": "Check if you have provided a username and a report",
-    "msg-45": "A report was sent to the team with {{email}} in copy"
+    "msg-1": "申请认证之前，你必须先接受我们的《学术诚信条例》",
+    "msg-2": "出现了一些奇怪的情况。如果再出现这种情况，请考虑在 https://github.com/freeCodeCamp/freeCodeCamp/issues/new 提 issue",
```

```suggestion
    "msg-2": "出现了一些奇怪的情况。如果再出现这种情况，请考虑在 https://github.com/freeCodeCamp/freeCodeCamp/issues/new 提",
```

Doesn't seem right, not sure.

---

It's ok to use "issue" directly in the translation. And it shouldn't be omitted.